docs/scala/support.md

```diff
@@ -157,56 +157,56 @@ This section lists the SDK types and functions currently supported in this versi
 ### Char
 | Function      | Morphir-Elm Version Introduced | Supported |
 |---------------|--------------------------------|-----------|
-| isUpper       | v2.31.3                        | &#x2715;  |
-| isLower       | v2.31.3                        | &#x2715;  | 
-| isAlpha       | v2.31.3                        | &#x2715;  | 
-| isAlphaNum    | v2.31.3                        | &#x2715;  | 
-| isDigit       | v2.31.3                        | &#x2715;  |
-| isOctDigit    | v2.31.3                        | &#x2715;  | 
-| isHexDigit    | v2.31.3                        | &#x2715;  | 
-| toUpper       | v2.31.3                        | &#x2715;  | 
-| toLower       | v2.31.3                        | &#x2715;  | 
-| toLocaleUpper | v2.31.3                        | &#x2715;  | 
-| toLocaleLower | v2.31.3                        | &#x2715;  | 
-| toCode        | v2.31.3                        | &#x2715;  | 
-| fromCode      | v2.31.3                        | &#x2715;  | 
+| isUpper       | v2.31.3                        | &#x2713;  |
+| isLower       | v2.31.3                        | &#x2713;  | 
+| isAlpha       | v2.31.3                        | &#x2713;  | 
+| isAlphaNum    | v2.31.3                        | &#x2713;  | 
+| isDigit       | v2.31.3                        | &#x2713;  |
+| isOctDigit    | v2.31.3                        | &#x2713;  | 
+| isHexDigit    | v2.31.3                        | &#x2713;  | 
+| toUpper       | v2.31.3                        | &#x2713;  | 
+| toLower       | v2.31.3                        | &#x2713;  | 
+| toLocaleUpper | v2.31.3                        | &#x2713;  | 
+| toLocaleLower | v2.31.3                        | &#x2713;  | 
+| toCode        | v2.31.3                        | &#x2713;  | 
+| fromCode      | v2.31.3                        | &#x2713;  | 
 
 ### Morphir.SDK.Decimal
 | Function          | Morphir-Elm Version Introduced | Supported |
 |-------------------|--------------------------------|-----------|
-| fromInt           | v2.8.0                         | &#x2713;  |
```

Comment:
Any concerns if we sort the table? Thought it be easier on eyes if the list is sorted.

Reply:
It's currently in the order that the IR functions are listed. Personally I don't think there's any need to sort it

Reply:
@michelchan Thanks for the clarification. I will revert the sort change.

Reply:
Reverted the result from sort.